### DISCUSSION TITLE
STM32 boards fail to install in Arduino IDE

### DISCUSSION THREAD

**Discussion by mromine3** [stm32duino/Arduino_Core_STM32#2290] [Q&A]
When I go to install "STM32 MCU based boards" on the arduino,IDE, I get the following error:

error: 13 INTERNAL: Cannot install tool STMicroelectronics:xpack-arm-none-eabi-gcc@12.2.1-1.2: testing local archive integrity: testing archive size: fetched archive size differs from size specified in index: 264812496 != 259513877

Can anyone help?

**Comment by fpistm** [comment]
Probably the archive was not properly downloaded and is corrupted. You should remove the one downloaded by the Arduino IDE in the staging folder.
On Windows it is on <USER HOME>\AppData\Local\Arduino15\staging\packages and on Linux $HOME/.arduino15/staging/packages

**Reply by mromine3** [reply]
Thanks for the great suggestion.  I tried this with no luck. 
I went a few steps further and uninstalled Arduino completely and blew away all file folders related to the Arduino IDE, and then did a fresh install.  Still didn't work.    Same error.  
To verify this was limited to my dev PC I ran a fresh install on a different PC and installed the STM32 board manager package with no issues.  So this does seem to be limited to one PC.  
Any ideas on what else might be causing a conflict with the STM32 board package on my dev PC? 


 From there I uninstalled the Arduino IDE

**Reply by fpistm** [reply]
Proxy issue? Antiviral? .... hard to tell.

**Reply by fpistm** [reply]
You can download the package and copy it to the staging folder
